pennylane/plugins/default_qubit.py

```diff
@@ -20,12 +20,15 @@
 simulation of a qubit-based quantum circuit architecture.
 """
 import itertools
+import functools
+from string import ascii_letters as ABC
```

Comment:
limits `default.qubit` to 26 wires minus the max number of wires used by any gate :smiley:

Reply:
Effectively to 24 wires, as we only use einsum for 1 and 2 qubit gates. I figured that anyone trying to simulate more than that would not want to use default qubit anyways.